### PR TITLE
add loneTeen

### PR DESCRIPTION
**Pull request by Tatenda-D-Mawoneke**

### CODE REVIEW

File: Warmup-1/loneTeen/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export function loneTeen(givenInput1: number, givenInput2: number): boolean {
```
Comment (JustinGuedes) [inline Warmup-1/loneTeen/src/index.ts:1]: `givenInput`? Strange name, could you please name it to something more descriptive

File: Warmup-1/loneTeen/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  const givenInput1IsTeen = givenInput1 >= 13 && givenInput1 <= 19;
+  const givenInput2IsTeen = givenInput2 >= 13 && givenInput2 <= 19;
+
+  return (givenInput1IsTeen && !givenInput2IsTeen) ||
```
Comment (JustinGuedes) [inline Warmup-1/loneTeen/src/index.ts:5]: Can you simplify this please

**Review by JustinGuedes — CHANGES_REQUESTED** [review]

File: Warmup-1/loneTeen/src/index.ts (hunk 2 above)
Comment (Tatenda-D-Mawoneke) [reply Warmup-1/loneTeen/src/index.ts]: Relatively.

File: Warmup-1/loneTeen/src/index.ts (hunk 1 above)
Comment (Tatenda-D-Mawoneke) [reply Warmup-1/loneTeen/src/index.ts]: Renamed accordingly.

File: Warmup-1/loneTeen/test/index.test.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+// We'll say that a number is "teen" if it is in the range 13..19 inclusive. Given 2 int values, return true if one or the other is teen, but not both.
+
+import { assert } from "chai";
+import { loneTeen } from "../src";
+
+// loneTeen(13, 99) → true
+// loneTeen(21, 19) → true
+// loneTeen(13, 13) → false
+
+describe("loneTeen", () => {
```
Comment (JustinGuedes) [inline Warmup-1/loneTeen/test/index.test.ts:10]: Please add a test when they're both not teens

File: Warmup-1/loneTeen/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+export function loneTeen(age1: number, age2: number): boolean {
+  const age1IsTeen = isTeenChecker(age1);
+  const age2IsTeen = isTeenChecker(age2);
+
+  return age1IsTeen && age2IsTeen ? false : true;
```
Comment (JustinGuedes) [inline Warmup-1/loneTeen/src/index.ts:5]: Simplify please

**Review by JustinGuedes — CHANGES_REQUESTED** [review]

File: Warmup-1/loneTeen/test/index.test.ts (hunk 3 above)
Comment (Tatenda-D-Mawoneke) [reply Warmup-1/loneTeen/test/index.test.ts:10]: Sure.

File: Warmup-1/loneTeen/src/index.ts (hunk 4 above)
Comment (Tatenda-D-Mawoneke) [reply Warmup-1/loneTeen/src/index.ts:5]: Sure.

**Review by JustinGuedes — APPROVED** [review]

**Pull request merged**